Language edits to smooth the general text

EDRi-org opened the pull request:
I have updated the English version for the Privacy4Kids aka Digital Defenders booklet today. Please have a look at the pull request and tell me what you think. I made several smaller language edits to the text but more importantly updated the recommended apps section at the end to reflect the tech change since. Happy to discuss and finalise it together.

EDRi-org committed:
Update en.txt

petterreinholdtsen commented:
[European Digital Rights]
> I have updated the English version for the Privacy4Kids aka Digital
> Defenders booklet today. Please have a look at the pull request and
> tell me what you think. I made several smaller language edits to the
> text but more importantly updated the recommended apps section at the
> end to reflect the tech change since. Happy to discuss and finalise it
> together.

If possible, can you please split the change into two pull requests, one
for the textual changes and another for the changes to the list of
recommended apps?

I believe the two sets of changes deserve separate and unrelated
discussions. :)

-- 
Happy hacking
Petter Reinholdtsen

EDRi-org committed:
Language edits to smooth the general text

This commit only contains smaller textual changes that should make the text more readable for kids (excl. chapter 10 on "tools we like")

EDRi-org committed:
Language edits to smooth the general text

This commit only contains smaller textual changes that should make the text more readable for kids (excl. chapter 10 on "tools we like")

EDRi-org commented:
> If possible, can you please split the change into two pull requests, one for the textual changes and another for the changes to the list of recommended apps?

Done. I hope I haven't messed up things too much. I'm still learning to use git :)

petterreinholdtsen commented on original/en.txt:
```diff
@@ -501,17 +494,17 @@ It is not the only app we can use to communicate safely, but it is one of the ea
 
 » QUEST 6: I SHOULD CHOOSE AS MY PASSWORD …
-[R] …a random combination of letters and special characters.
+[R] …a random combination of letters, numbers and special characters.
```
Is 'digits' better than 'numbers'?

EDRi-org replied:
Good question. I used "numbers" because that's how Keepass calls it.

petterreinholdtsen commented:
Just for the record, I am no native English speaker, and not really capable of evaluating this change.
